### DISCUSSION TITLE
Remove Hermes (amphibious SAM launcher)

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#130] [Ideas]
Similar reasoning to the Walrus.

The Longbow serves a clear purpose of having omni, with the SAM launcher just being a secondary system.
The Cougar is quite interesting and I'd like to see it get buffed a bit, while the Banisher could do with a nerf.

If you, as a UEF player want a radar with you on the water, then I think you should have to use spy planes or navy (or ACU/sACU intel upgrades)

**Comment by PurpleMossClump** [comment]
I think it's just another case of having an amphibious unit of every class at T3, which we've discussed in another topic is not necessary. 

If not having purely standard radar on a T3 AA unit becomes an issue for UEF players, we could throw a small radar on the Cougar or Banisher.
